DOCS-2673: Change authentication method for API onboarding requests (#1907)
MultiSafepay/docs: content/my-account/partner-account/api-onboarding/add-ubos.md

    "document_type": "id",
    "filename": "identity-of-ubo.pdf",
    "id": "agi6ehoreex6a",
    "merchant_id": {affiliate_account_id},
    "mime_type": "application/pdf",
    "ubo_id": "{ubo_id}"
  },
  "success": true
}
```
{{< /collapse >}}

---

## 6. List identity documents

`GET` `https://testapi.multisafepay.com/v1/json/ubos/{ubo_id}/identitydocs?api_key={your-account-api-key}`

> Retrieve an array of all identity documents linked to a UBO.

Reviewer: ```suggestion
Retrieve an array of all identity documents for a UBO.
```
